tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/SDKClasspathContainerInitializer.java

```diff
@@ -77,7 +77,9 @@ public void initialize( IPath containerPath, IJavaProject project ) throws CoreE
         if ( bundle == null )
         {
             final String msg = "Invalid sdk properties setting.";
-            throw new CoreException( ProjectCore.createErrorStatus( msg ) );
+            //throw new CoreException( ProjectCore.createWarningStatus( msg ) );
```

Comment:
we should not suppress this exception, user wont see if it we don't throw coreException.